tests/transform/test_modelAD_general_transform.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import pandas as pd
+import pytest
+
+from agoradatatools.etl.transform.modelAD_general_transform import (
+    modelAD_general_transform,
+)
+
+
+class TestTransformGeneralModelAD:
```

Comment:
Are we specifically testing some `biomarkers` data and some `pathology` data? If not do you think we should?

Reply:
The source data should be consistent across the two datasets, so I don't think this is needed.